docs/installation.rst

```diff
@@ -97,14 +97,20 @@ You will need:
 
     - A pythonanywhere.com account (Free version works fine)
 
-Uploading the bot's files to Pythonanywhere
--------------------------------------------
-
-#. Download the bot's code from `this link <https://github.com/Mikadily/poloniexlendingbot/archive/master.zip>`_.
-#. Extract the files, and run ``lendingbot.py`` once to generate the default.cfg
-#. Modify the default.cfg with your settings (See Configuration.)
-#. Go to the "files" tab of the website.
-#. Create a new directory, preferably named "poloniexlendingbot" and upload all the files within the folder. (You cannot upload the .zip or a folder itself, you must do all the contents.)
+Downloading the bot's files to Pythonanywhere
+---------------------------------------------
+
+#. Start a new ``bash`` console from the "Consoles" tab.
+#. Get the source code from git GitHub by running ``git clone https://github.com/Mikadily/poloniexlendingbot``.
+#. You should see some output with couters increasing.
```

Comment:
I'm getting quite bad for spelling recently. Is there a bot that can check these for us?